modules/installation-configuration-parameters.adoc

```diff
@@ -474,15 +474,15 @@ Optional installation configuration parameters are described in the following ta
 |String
 
 |`capabilities`
-|Controls the installation of optional core cluster components. You can reduce the footprint of your {product-title} cluster by disabling optional components.
+|Controls the installation of optional core cluster components. You can reduce the footprint of your {product-title} cluster by disabling optional components. For more information, see the "Cluster capabilities" page in _Installing_.
 |String array
 
 |`capabilities.baselineCapabilitySet`
-|Selects an initial set of optional capabilities to enable. Valid values are `None`, `v4.11` and `vCurrent`. `v4.11` enables the `baremetal` Operator, the `marketplace` Operator, and the `openshift-samples` content. `vCurrent` installs the recommended set of capabilities for the current version of {product-title}. The default value is `vCurrent`.
+|Selects an initial set of optional capabilities to enable. Valid values are `None`, `v4.11`, `v4.12` and `vCurrent`. `v4.11` enables the `baremetal` Operator, the `marketplace` Operator, and the `openshift-samples` content. `v4.12` includes the `v4.11` set as well as `Console`, `Insights`, `CSISnapshot` and `Storage`. `vCurrent` installs the recommended set of capabilities for the current version of {product-title}. The default value is `vCurrent`.
 |String
 
 |`capabilities.additionalEnabledCapabilities`
-|Extends the set of optional capabilities beyond what you specify in `baselineCapabilitySet`. Valid values are `baremetal`, `marketplace` and `openshift-samples`. You may specify multiple capabilities in this parameter.
```

Comment:
It feels a bit strange to me to have an explicit list of possible capability sets above _and_ break those down into constituent capability slugs.  But then to punt out to external docs here for the overall set of capability slugs.  I'm ok with the inconsistent approach if that feels better to docs-y folks, but I'd have expected either "punt all the alternative-option listings out to the external caps docs" or "inline all the slugs here, and punt out to the caps docs for the semantics behind each slug".

Reply:
You're probably right about that - this table shouldn't have each cap explained since the user can be expected to find this page *after* the cluster caps page, and we have a reference to the full doc on line 477.